### PR TITLE
Changes Update in Contribution.md

### PR DESCRIPTION
**Pull request by i-am-SnehaChauhan**
Now, please check and merge this branch. Thankyou

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -31,14 +31,65 @@ If you're new to open source and don't know where to start, don't worry! We've t
...
-- Test the changes you have made using ```npm run dev``` or ```npm run dev -- --host```.
-- Commit your changes and push them to your forked repository.
-- Submit a pull request to the main project repository.
+There are some steps that you need to follow before making any contribution in this project, the following steps are important to get start.
+
+### Step 1. Fork the Repository
+ [Fork](https://github.com/ajaynegi45/Library.git) the repo first
+
+### Step 2. Clone the forked Repository 
+    ```
```
Comment (ajaynegi45) [inline contributing.md:40]: These ``` are also printing

File: contributing.md [hunk 2]
```diff
@@ -31,14 +31,65 @@ If you're new to open source and don't know where to start, don't worry! We've t
...
+git checkout -b <your_branch_name>
+```
+## Step 7. Making Changes
+Perform your desired changes to the code base.
+Test the changes you have made using 
+```npm run dev``` or ```npm run dev -- --host```.
+
+
+### Step 8. Track Changes
+    ```
```
Comment (ajaynegi45) [inline contributing.md:72]: These ``` are also printing

File: contributing.md [hunk 3]
```diff
@@ -31,14 +31,65 @@ If you're new to open source and don't know where to start, don't worry! We've t
...
+     # Track the changes
+     git status
+
+     # Add changes to Index
+     git add . or git add <file_name>
+     ```
+### Step 9. Commit Changes
+    ```
```
Comment (ajaynegi45) [inline contributing.md:80]: These ``` are also printing

File: contributing.md [hunk 4]
```diff
@@ -31,14 +31,65 @@ If you're new to open source and don't know where to start, don't worry! We've t
...
+
+
+### Step 8. Track Changes
+    ```
+     # Track the changes
+     git status
+
+     # Add changes to Index
+     git add . or git add <file_name>
+     ```
```
Comment (ajaynegi45) [inline contributing.md:78]: These ``` are also printing

File: contributing.md [hunk 5]
```diff
@@ -31,14 +31,65 @@ If you're new to open source and don't know where to start, don't worry! We've t
...
+### Step 9. Commit Changes
+    ```
+    git commit -m "Write a meaningful but small commit message" 
+    ```
```
Comment (ajaynegi45) [inline contributing.md:82]: These ``` are also printing

File: contributing.md (hunk 2 above)
Comment (i-am-SnehaChauhan) [reply contributing.md:72]: Okay i am changing this

File: contributing.md [hunk 6]
```diff
@@ -31,14 +31,65 @@ If you're new to open source and don't know where to start, don't worry! We've t
...
+    git clone https://github.com/<your_github_username>/Library.git
+    ```
+### Step 3. Set Up Remotes 
+- Add a reference (remote) to the original repository:
+    ```
+    git remote add upstream https://github.com/ajaynegi45/Library.git
+    ``` 
+    - Add a reference (remote) to your forked repository:
+```
+git remote add origin https://github.com/<your_user_name>/Library.git
```
Comment (ajaynegi45) [inline contributing.md:50]: Align both point and code properly

Screenshot 2023-10-06 at 8 52 47 PM

**Review by ajaynegi45 — CHANGES_REQUESTED** [review]
Resolve these issues and create new PR

File: contributing.md (hunk 6 above)
Comment (i-am-SnehaChauhan) [reply contributing.md:50]: okay

**Pull request closed without merging**